chainregistry/chainparams.go

```diff
@@ -42,7 +42,7 @@ var BitcoinMainNetParams = BitcoinNetParams{
 	CoinType: keychain.CoinTypeBitcoin,
 }
 
-// BitcoinSimNetParams contains parameters specific to the simulation test
+// BitcoinSimNetParams contains parameters specific to the specific test
```

Comment:
hm, why change this?

Reply:
I type out the code moves so I saw "specific" previously and I guess I copied it instead of "simulation" o.o